vortex-array/src/arrays/varbin/compute/is_constant.rs

```diff
@@ -14,3 +13,15 @@ impl IsConstantFn<&VarBinArray> for VarBinEncoding {
         array.with_iterator(compute_is_constant).map(Some)
     }
 }
+
+pub(super) fn compute_is_constant(iter: &mut dyn Iterator<Item = Option<&[u8]>>) -> bool {
+    let Some(first_value) = iter.next() else {
+        return true; // empty array is constant
```

Comment:
I think this should be false. And document it in IsConstantFn docstring

Reply:
we take care of that in the entry function, but no harm doing it here correctly.